lib/block-supports/layout.php

```diff
@@ -123,9 +130,7 @@ function gutenberg_get_layout_style( $selector, $layout, $has_block_gap_support
 				$gap_value  = $gap_row === $gap_column ? $gap_row : $gap_row . ' ' . $gap_column;
 			}
 			if ( $gap_value && ! $should_skip_gap_serialization ) {
-				$style .= "$selector {";
-				$style .= "gap: $gap_value;";
-				$style .= '}';
+				$layout_styles[ $selector ] = array( 'gap' => $gap_value );
```

Comment:
This rule is overwriting the `flex-wrap` rule set above.

Reply:
fixed